wormhole/wormhole.py

```
@@ -59,12 +70,22 @@ async def on_message_without_command(self, message: discord.Message):
             return  # Ignore bot commands
         
         linked_channels = await self.config.linked_channels_list()
+        guild = message.guild
+        author = message.author
         if message.channel.id in linked_channels:
             for channel_id in linked_channels:
                 if channel_id != message.channel.id:
                     channel = self.bot.get_channel(channel_id)
                     if channel:
-                        await channel.send(f"**{message.author.display_name}:** {message.content}")
+                        member = guild.get_member(author.id)
+                        if member:
+                            display_name = author.display_name if author.display_name else author.name
+                            rank_id = await self.config.guild(guild).get_raw(f"member_{author.id}")
```

6 | #734 [2024-06-15 16:29:00] ERROR [discord.client] Ignoring exception in on_message_without_command.
7 | Traceback (most recent call last):
8 | File "/data/venv/lib/python3.11/site-packages/discord/client.py", line 441, in _run_event
9 | await coro(*args, **kwargs)
10 | File "/data/cogs/CogManager/cogs/wormhole/wormhole.py", line 83, in on_message_without_command
11 | rank_id = await self.config.guild(guild).get_raw(f"member_{author.id}")
12 | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
13 | File "/data/venv/lib/python3.11/site-packages/redbot/core{HOME}.py", line 514, in get_raw
14 | raw = await self._driver.get(identifier_data)
15 | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
16 | File "/data/venv/lib/python3.11/site-packages/redbot/core/_drivers/json.py", line 138, in get
17 | partial = partial[i]
18 | ~~~~~~~^^^
19 | KeyError: 'GUILD'